pkg/collector/corechecks/oracle-dbm/custom_queries.go

```diff
@@ -65,15 +69,16 @@ func (c *Check) CustomQueries() error {
 		"histogram":       sender.Histogram,
 		"historate":       sender.Historate,
 	}
+	var allErrors error
```

Comment:
I think the joining of multiple errors might be a good use case for the new (as of go 1.20) [errors.Join](https://pkg.go.dev/errors#Join).

Reply:
That's what I implemented in the first place, but the next custom Oracle build has to be made on top of `7.47.0`, which has to be compiled with an earlier `Go` version. We can change this, once `7.48` is out.

Reply:
ah, makes sense. 💯